### PR TITLE
rename MP2 CMake option to MP2_ENCODING

### PR DESCRIPTION
**Pull request by Be-ing**
Signed-off-by: Be <be@mixxx.org>

Resolves: Twolame only does encoding, not decoding, of MP2 audio.

<details>
<summary>Checklist</summary>

- [x] I have signed off my commits using `-s` or `Signed-off-by`\* (See: [Contributing § DCO](https://github.com/tenacityteam/tenacity/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))
- [x] I made sure the code compiles on my machine
- [x] I made sure there are no unnecessary changes in the code\*
- [x] I made sure the title of the PR reflects the core meaning of the issue you are solving\*
- [x] I made sure the commit message(s) contain a description and answer the question "Why do those changes fix that particular issue?" or "Why are those changes really necessary as improvements?"\*

\* indicates required

</details>

### CODE REVIEW

File: BUILDING.md [hunk 1]
```diff
@@ -60,9 +60,9 @@ Then install Tenacity's dependencies:
 sudo dnf install alsa-lib-devel cmake expat-devel flac-devel gcc-g++ gettext-devel lame-devel libid3tag-devel libmad-devel libogg-devel libsndfile-devel libvorbis-devel lilv-devel lv2-devel portaudio-devel portmidi-devel serd-devel sord-devel soundtouch-devel soxr-devel sqlite-devel sratom-devel suil-devel taglib-devel twolame-devel vamp-plugin-sdk-devel wxGTK-devel zlib-devel ccache ninja-build git ffmpeg-devel
 ```
 
-If you use a high DPI screen, the wxWidgets 3.1.4 package in Fedora does
-not work well for that. You can compile wxWidgets 3.1.5 instead of using
-the Fedora package. Refer to the
+If you use a high DPI screen, the wxWidgets 3.1.4 package in Fedora 34
+does not work well for that. You can compile wxWidgets 3.1.5 instead of
+using the Fedora package. Refer to the
```
Comment (Semisol) [inline BUILDING.md]: This is not relevant..? Please make another PR, or split it into its own commit & rename the PR and edit the desc to account for it

**Review by Semisol — CHANGES_REQUESTED** [review]
Looks good to me but...

File: BUILDING.md (hunk 1 above)
Comment (Be-ing) [reply BUILDING.md]: whoops, I did not mean to commit those in this branch

Comment (Semisol) [reply BUILDING.md]: okay, makes sense

**Review by Semisol — APPROVED** [review]
